new-cdc/eventuate-local-java-cdc-connector-unified/src/main/java/io/eventuate/local/unified/cdc/pipeline/common/properties/RawUnifiedCdcProperties.java

```diff
@@ -0,0 +1,36 @@
+package io.eventuate.local.unified.cdc.pipeline.common.properties;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+import java.util.Map;
+
+@ConfigurationProperties(prefix = "eventuate.cdc")
+public class RawUnifiedCdcProperties {
+  private Map<String, Map<String, Object>> reader;
```

Comment:
Why `Map<String, Object>` and not `Map<String, String>`

Reply:
It seems, there is no difference